### PR TITLE
#131356235 Czar Draws Card

### PR DESCRIPTION
**Pull request by andela-aatanda**
#### What does this PR do?

Adds feature that allows Czar to draw cards rather than been random

#### Description of Task to be completed?

Given a User is the Czar, User should be able to click a deck to transition in question for new round

#### How should this be manually tested?

Visit deployed app for this PR [here](https://hyuga-cfh-staging-pr-45.herokuapp.com/#!/)

#### What are the relevant pivotal tracker stories?

Story id: 131356235

### CODE REVIEW

File: app/views/includes/foot.jade [hunk 1]
```diff
@@ -43,6 +43,9 @@ script(type='text/javascript', src='/js/controllers/header.js')
 script(type='text/javascript', src='/js/controllers/game.js')
 script(type='text/javascript', src='/js/init.js')
 
+// React bundle
+script(type='text/javascript', src='/js/bundle.js')
+
```
Comment (azemoh) [inline app/views/includes/foot.jade]: @andela-aatanda we don't need react bundle `bundle.js`  anymore.

File: gulpfile.js [hunk 2]
```diff
@@ -60,6 +60,14 @@ gulp.task('bower', function () {
     .pipe(gulp.dest('public/lib'));
 });
 
+// Webpack task
+gulp.task('webpack', function (done) {
```
Comment (azemoh) [inline gulpfile.js]: We also do not need the Webpack gulp task. It has been removed from the codebase.

File: package.json [hunk 3]
```diff
@@ -64,6 +64,8 @@
     "passport-google-oauth": "~0.1.5",
     "passport-local": "~0.1.6",
     "passport-twitter": "~1.0.2",
+    "react": "^15.3.2",
+    "react-dom": "^15.3.2",
```
Comment (azemoh) [inline package.json]: No need for `react` & `react-dom`.

File: webpack.config.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
+var debug = process.env.NODE_ENV !== 'production';
```
Comment (azemoh) [inline webpack.config.js]: `webpack.config.js` have also been removed from the project.

File: config/socket/game.js [hunk 5]
```diff
@@ -36,8 +36,8 @@ function Game(gameID, io) {
   this.curQuestion = null;
   this.timeLimits = {
     stateChoosing: 21,
-    stateJudging: 16,
-    stateResults: 6
+    stateJudging: 16
+    // stateResults: 6
```
Comment (andela-koni) [inline config/socket/game.js]: Please remove commented out codes

File: config/socket/game.js [hunk 6]
```diff
@@ -215,10 +214,12 @@ Game.prototype.stateResults = function(self) {
...
     if (winner !== -1) {
       self.stateEndGame(winner);
-    } else {
-      self.stateChoosing(self);
     }
-  }, self.timeLimits.stateResults*1000);
+  }, self.timeLimits.stateResults * 1000);
+};
+
+Game.prototype.beginNextRound = function (self) {
```
Comment (andela-koni) [inline config/socket/game.js]: remove space between function and the opening bracket

**Pull request merged**